x/oracle/types/keys.go

```diff
@@ -21,45 +21,40 @@ const (
 
 // KVStore key prefixes
 var (
-	KeyPrefixExchangeRate                 = []byte{0x01} // prefix for each key to a rate
-	KeyPrefixFeederDelegation             = []byte{0x02} // prefix for each key to a feeder delegation
-	KeyPrefixMissCounter                  = []byte{0x03} // prefix for each key to a miss counter
-	KeyPrefixAggregateExchangeRatePrevote = []byte{0x04} // prefix for each key to a aggregate prevote
-	KeyPrefixAggregateExchangeRateVote    = []byte{0x05} // prefix for each key to a aggregate vote
-	ExchangeRateKey                       = []byte{0x01} // prefix for each key to a rate
-	FeederDelegationKey                   = []byte{0x02} // prefix for each key to a feeder delegation
-	MissCounterKey                        = []byte{0x03} // prefix for each key to a miss counter
-	AggregateExchangeRatePrevoteKey       = []byte{0x04} // prefix for each key to a aggregate prevote
-	AggregateExchangeRateVoteKey          = []byte{0x05} // prefix for each key to a aggregate vote
+	ExchangeRateKey                 = []byte{0x01} // prefix for each key to a rate
```

Comment:
We should still keep the `KeyPrefix*` prefixes and remove the `Key` suffix IMO. This way, we're consistent with `x/leverage` as well.

Reply:
It reflects the context of what it's used for. E.g. for errors that you'd wrap, it'd be `Err*`.

Reply:
Yeah that's fair 👍